CMakeLists.txt

```diff
@@ -69,6 +69,9 @@ IF(EIGEN_NO_AUTOMATIC_RESIZING)
   ADD_DEFINITIONS(-DEIGEN_NO_AUTOMATIC_RESIZING)
 ENDIF(EIGEN_NO_AUTOMATIC_RESIZING)
 
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
This actually limit the standard to C++11, even if most modern compilers default to C++14, so this could raise some issues. Could you please use `CHECK_MINIMAL_CXX_STANDARD(11 ENFORCE)` ?

This is not a standard CMake macro, but it is defined in a submodule included in this project:
https://github.com/jrl-umi3218/jrl-cmakemodules/blob/master/cxx-standard.cmake